### PR TITLE
Add tutorial for warehouse_ros(_sqlite)

### PR DESCRIPTION
**Pull request by gleichdick**
### Description

See ros-planning/moveit2_tutorials#101

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/persistent_scenes_and_states/persistent_scenes_and_states.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+You can find the launch file :codedir:`here in the tutorials GitHub repository <persistent_scenes_and_states/warehouse_settings.launch.xml>`
+or (unmodified) in your workspace at ``install/share/moveit_resources_panda_moveit_config/launch/warehouse_settings.launch.xml``.
+
+Connecting to the storage backend
+---------------------------------
+
+After choosing the storage plugin and configuring the launch file(s),
+run RViz using ::
+
+   roslaunch moveit_resources_panda_moveit_config demo.launch db:=true
```
Comment (gleichdick) [inline doc/persistent_scenes_and_states/persistent_scenes_and_states.rst:41]: `panda_moveit_config` seems to be broken, so I'm using `moveit_resources_panda_moveit_config` instead...

File: doc/persistent_scenes_and_states/warehouse.launch [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<launch>
+
+  <!-- The path to the database must be specified -->
+  <arg name="moveit_warehouse_database_path" />
+
+  <!-- Load warehouse parameters -->
+  <include file="$(find moveit_resources_panda_moveit_config)/launch/warehouse_settings.launch.xml" />
+
+## BEGIN_TUTORIAL
+## Optionally, start the MongoDB Server (uncomment the follwing in ``warehouse.launch``)
```
Comment (v4hn) [inline doc/persistent_scenes_and_states/warehouse.launch:10]:
```suggestion
## Optionally, start the MongoDB Server (uncomment the following in ``warehouse.launch``)
```

**Review by v4hn — APPROVED** [review]

**Comment by gleichdick** [comment]
@v4hn friendly ping

**Comment by v4hn** [comment]
> v4hn friendly ping

Sorry, I was hoping someone else would pick it up after my review, but apparently the interest is relatively low compared to other hot topics :-) I added some more patches to the tutorial and fixed an issue in the MoveIt display that you probably encountered as well while you tested.

I still think to be usable and really useful this functionality needs almost an entire cleanup in the setup assistant / the config templates and the RViz display integration...

I will merge this now to move forward.

As discussed before, could you transfer the repository to the `ros-planning` organization then and report back in a corresponding issue? I believe @davetcoleman (@henningkayser please talk to him in person if he shouldn't happen to read this thread :sunglasses: ) needs to accept the transfer and might have to add him back as owner for the repository afterwards.
I've never been involved in such a repository transfer, so I'm unsure how this is implemented.

**Pull request merged**